File: Python/ExerciciosPython/Exercicio10.py
# Algoritmo que lê um número e verifica se ele é par ou ímpar. 
# Quando for par, armazena em valor 'p' e quando for ímpar, armazena em 'i'. 
# Exibir 'p' e 'i' no final do processamento.

# Variáveis
p = 0
i = 0

# Entrada
num = int(input("Insira um número: "))

# Processamento
if num % 2 == 0:
    p = num
else:
    i = num

# Saída
print(f"Número par: {p}")
print(f"Número ímpar: {i}")

File: C/ExercíciosC/codigoFonte/Exercicio10.c
//Algoritmo que lê um número e verifica se ele é par ou ímpar.
//Quando for par, armazenar em 'p', se for ímpar, armazenar em i.
//Exibe 'p' e 'i' no final do processamento.
#include <stdio.h>

int main() {
    //Variáveis
    int num, p = 0, i = 0;

    //Entrada
    printf("Insira um número: \n");
    scanf("%d", &num);

    //Processamento
    if (num % 2 == 0) { //% é módulo
        p = num;
    }else{
        i = num;
    }
    //Saída
    printf("%d\n", p);
    printf("%d\n", i);
    return 0;
}